packages/@lwc/wds-playground/src/browser/ui/pg/playground/playground.js

```diff
@@ -67,6 +120,10 @@ export default class Playground extends LightningElement {
 
   onConfigChange(evt) {
     this.config = evt.detail;
+
+    if (this.config.observeLayoutShift) {
```

Comment:
It looks as though layout shifts detection is initially toggled on, it never goes off again. I think the `onCLS` approach needs to be replaced with using the browser APIs directly with `PerformanceObserver`.

Reply:
done, using native `PerformanceObserver ` API now